project/profiles/capacity4more/modules/c4m/features_og/c4m_features_og_events/c4m_features_og_events.module

```diff
@@ -162,7 +162,7 @@ function c4m_features_og_events_upcoming_events_loggedin() {
 
   $user_wrapper = entity_metadata_wrapper('user', $user->uid);
 
-  // Create empty array for both $interests & $groups.
+  // Initiate $interests and $groups
```

Comment:
@guytepper 
the sniffer will fail in here since you dropped the `.` from the end of the comment.